api/src/main/java/org/openmrs/module/ModuleUtil.java

```diff
@@ -440,10 +440,10 @@ public static int compareVersion(String version, String value) {
 			
 			// match the sizes of the lists
 			while (versions.size() < values.size()) {
-				versions.add("0");
+				versions.add("1.8.4-SNAPSHOT");
```

Comment:
what were you trying to do here??

Reply:
Thought that would set snapshot older by changing that line